docs/en/user_guide.md

```diff
@@ -0,0 +1,78 @@
+
+**Border header** - Toggle a class to allow for an alternate border theme for the table header. Uncheck and it will use the default element styles, check to apply the class styles (if any are defined in the site theme css).
+
+**Header Vertical Alignment** - Set the vertical alignment of the table header cell content.
+
+**Header Horizontal Alignment** - Set the horizontal alignment of the table header cell content.
+
+**Last row is footer** - Enable this to set the last row (last Table Item) as the table footer (`<tfoot>`).
+
+**Border footer** - Toggle a class to allow for an alternate border theme for the table footer. Uncheck and it will use the default element styles, check to apply the class styles (if any are implemented).
+
+**Footer Vertical Alignment** - Set the vertical alignment of the table footer cell content.
+
+**Footer Horizontal Alignment** - Set the horizontal alignment of the table footer cell content.
+
+**First column is header** - Turn the first cell in each row into a `<th>`. Will also alter the content of table cells at smaller screen sizes.
+
+> Take care enabling both "First Column is header" and "First row is header", as it can result in confusing table markup and\or rendering if there are not enough rows and columns available to make sense, or the styles are not available to differentiate effectively between `<th>` and `<td>`.
+
+**Border first column** - Toggle a class to allow for an alternate border theme for the table `<th>` cells (default element style or special class style).
+
+**Header Column Vertical Alignment** - Set the vertical alignment of the table body `<th>` cell content.
+
+**Header Column Horizontal Alignment** - Set the horizontal alignment of the table body `<th>` cell content.
+
+**Border Outer** - Toggle a class to allow for an alternate border theme around the outside of the table.
+
+**Border rows** - Toggle a class to allow for an alternate horizontal border theme inside the table.
+
+**Border cols** - Toggle a class to allow for an alternate vertical border theme inside the table.
+
+**Zebra rows** - Toggle a class to allow for an alternate theme on every second row in the table (often used for background shading, but may include other attributes as determined by the theme css).
+
+> Each of these toggles require a set of corresponding class styles to be effective.
+
+**Cell Vertical Alignment** - Set the vertical alignment of the table `<td>` cell content.
+
+**Cell Horizontal Alignment** - Set the horizontal alignment of the table `<td>` cell content.
+
+**Proportion allocated to Column** - Set a percentage width for the column.  There will be one field per column (as defined by the Number of Columns field in the Content tab). These values should sum to 100%.
+
+> If over, the earlier columns will assume priority. If under, the browser will attempt to distribute as evenly as possible while respecting the larger proportions.
```

Comment:
Is this code, or describing observed browser behaviour? If the latter, is there a standard describing this behaviour?